lib/packwerk/cli.rb

```diff
@@ -11,6 +9,20 @@ class Cli
 
     autoload :Result
 
+    class << self
+      extend T::Sig
+
+      sig { params(command: String).returns(T.nilable(T.class_of(Commands::BaseCommand))) }
+      def command_class_for(command)
+        command_class_name = command.sub(" ", "_").underscore.classify + "Command"
+        if Commands.const_defined?(command_class_name)
+          Commands.const_get(command_class_name) # rubocop:disable Sorbet/ConstantsFromStrings
```

Comment:
I tried with the registry pattern first, but it required me to `require` all the files (with `Dir[]` and a glob for example) preemptively.
Using `const_get`, we can keep lazy loading.